api/models.py

```diff
@@ -14,3 +14,22 @@ def __str__(self):
         """one-line docstring for representing the Profile object."""
         return f'{self.user.first_name}'
 
+
+
+
+
+class add_loan_record(models.Model):
+    loan=models.CharField(max_length=200, null=True)
```

Comment:
What's this loan field doing? I don't understand

Reply:
the is the amount of loan that the user will be agreeing on that it what it meant for